tests/unit_tests/Commands/Self/ConsoleCommandTest.php

```diff
@@ -28,6 +30,30 @@ public function setUp()
      */
     public function testConsole()
     {
+        if (version_compare(PHP_VERSION, '7.1.0') < 0) {
+            $out = $this->command->console('site.env');
+            $this->assertNull($out);
+        }
+    }
+
+    public function testConsoleIncompatibleMessage()
+    {
+        $config = $this->getMockBuilder(TerminusConfig::class)
```

Comment:
I'll use the incomplete for the first test, but this stays because I don't want my testing coverage to drop.